INSTALL.md

```diff
@@ -0,0 +1,62 @@
+## Installation instructions
+**Note: GCconnex does not work with PHP7 - GCconnex ne fonctionne pas avec PHP7**
+
+### Ubuntu 14.04
+#### Install Git, Apache, MySQL, PHP and libs
+    sudo aptitude install git apache2 mysql-server php5 libapache2-mod-php php5-mysql php5-gd
+When prompted, enter a root password for MySQL.
+
+#### Fork and Clone GCConnex Github Repo
+    git clone https://github.com/tbs-sct/gcconnex.git
+    git checkout gcconnex
+
+#### Create data directory
+    mkdir gcconnex_data
```

Comment:
Of course.  I was assuming starting in the home directory..  and gcconnex_data is not in gcconnex.  Then link only gcconnex in /var/www/html/.

What's confusing is ```git checkout gcconnex```, since you would have to be in gcconnex folder.